DOC-6607-Couchbase-Lite-2-7-1-Release-Notes

ibsoln opened the pull request:
https://issues.couchbase.com/browse/DOC-6607

This epic comprises the following tickets: 

- DOC-6608-Java 2.7.1 Rel Notes - https://issues.couchbase.com/browse/DOC-6608
- DOC-6609-Android 2.7.1 Rel Notes - https://issues.couchbase.com/browse/DOC-6609
- DOC-6610-Net 2.7.1 Rel Notes - https://issues.couchbase.com/browse/DOC-6610
- DOC-6611-ObjC 2.7.1 Rel Notes - https://issues.couchbase.com/browse/DOC-6611
- DOC-6612-Swift 2.7.1 Rel Notes - https://issues.couchbase.com/browse/DOC-6612

ibsoln committed:
DOC-6607-Couchbase-Lite-2-7-1-Release-Notes

https://issues.couchbase.com/browse/DOC-6607

DOC-6608-Java 2.7.1 Rel Notes
DOC-6609-Android 2.7.1 Rel Notes
DOC-6610-Net 2.7.1 Rel Notes
DOC-6612-ObjC 2.7.1 Rel Notes
DOC-6612-Swift 2.7.1 Rel Notes

DOC-6612-Swift 2.7.1 Rel Notes

DOC-6611-ObjC 2.7.1 RelNotes

DOC-6210 .Net 2.7.1 RelNotes

https://issues.couchbase.com/browse/DOC-6610

DOC-6609 Android 2.7.1 RelNotes

https://issues.couchbase.com/browse/DOC-6609

DOC-6608 Java 2.7.1 RelNotes

https://issues.couchbase.com/browse/DOC-6608

ibsoln commented:
@rajagp @djpongh @amarantha-k  Couchbase Lite 2.7.1 Release Notes for review etc.

amarantha-k reviewed (approved):
lgtm

djpongh reviewed (commented):
Need to add:
+ * https://issues.couchbase.com/browse/CBL-849[*CBL-849*] Returning WebProxy null when WINHTTP_PROXY_INFO out as invalid value if that ever happens (for .NET). I added comments per file

djpongh commented on modules/ROOT/pages/csharp.adoc:
```diff
@@ -1091,6 +1091,29 @@ The Couchbase Lite API is thread safe except for calls to mutable objects: `Muta
+==== {fixed}
+
+This maintenance release fixes the following issues:
+
```
Add https://issues.couchbase.com/browse/CBL-849 to .NET fixed section

ibsoln commented:
> Need to add:
> 
> * * [https://issues.couchbase.com/browse/CBL-849[*CBL-849*]](https://issues.couchbase.com/browse/CBL-849%5B*CBL-849*%5D) Returning WebProxy null when WINHTTP_PROXY_INFO out as invalid value if that ever happens

@djpongh Is that just for C#/.Net or across the board?

djpongh commented on modules/ROOT/pages/java-android.adoc:
```diff
@@ -1180,6 +1180,31 @@ The Couchbase Lite API is thread safe except for calls to mutable objects: `Muta
+
+This maintenance release fixes the following issues:
+
+* https://issues.couchbase.com/browse/CBL-789[*CBL-789*] Crash when accessing `connection->name()`
```
Needs: https://issues.couchbase.com/browse/CBL-799

djpongh commented on modules/ROOT/pages/java-platform.adoc:
```diff
@@ -1679,9 +1679,30 @@ Back to <<Preparing Your Build Environment>>
+==== {fixed}
+
+This maintenance release fixes the following issues:
+
```
Needs: https://issues.couchbase.com/browse/CBL-799

djpongh reviewed (commented):
Missed two issues, one was accidentally marked private (made it public) and the other has two milestones on it.

ibsoln committed:
DOC-6607-Post-review fixes

- https://issues.couchbase.com/browse/DOC-6607

- https://github.com/couchbase/docs-couchbase-lite/pull/206#pullrequestreview-397511134

ibsoln commented:
> Missed two issues, one was accidentally marked private (made it public) and the other has two milestones on it.

@djpongh @amarantha-k  Post-review fixes applied - ready to re-review.

Commit: https://github.com/couchbase/docs-couchbase-lite/pull/206/commits/14c813f5b0377d024d578c9111f0c28bfcd2816f
Staged: https://ibsoln.github.io/stage/DOC-6607-post-review/couchbase-lite/2.7/index.html
